configuration/drugs/1020_essential_drugs.csv

```diff
@@ -0,0 +1,835 @@
+53c3e8c7-475a-48da-a11a-e1c450c88ff3,0,Ferreux Sel+Folique Acide (66mg Fe2+0.25mg),Ferrous Salt + Folic Acid,Comprimé,66mg +0.25mg,,
+a16fae82-2dc7-457d-bffa-b5727ff278e2,0,Filgastim 30mg/ml,Filgastim,Injection,30mg/ml,,
+96a2890e-09eb-452c-a93c-f8ad803f1052,0,Fluconazole 200mg,Fluconazole,Comprimé,200mg,,
+5b33fa4d-2f12-4544-a431-7a4d69068c9f,0,Fluconazole 200mg/100ml,Fluconazole,Injection,200mg/100ml,,
+c9005cb5-8bb6-4aec-906b-2a071e91c882,0,Fluconazole 2mg/ml,Fluconazole,Injection,2mg/ml,,
+3a6a306a-567b-488a-8e81-266858d2a23e,0,Fluconazole 400mg,Fluconazole,Comprimé,400mg,,
+2502313c-197b-4901-938d-e3fb98eacd3d,0,Fluconazole 50mg,Fluconazole,Capsule,50mg,,
+71ddbcc4-fcc1-4c76-8320-8f5816eb17a9,0,Fluconazole 50mg/5ml f/60ml,Fluconazole,Sirop,50mg/5ml,,
+bc54109d-9d07-4317-a048-48d4d5f33b77,0,Fluconazole 50mg/5ml pps 50ml,Fluconazole,Comprimé,50mg/5ml,,
+2e8bcc5f-91f0-400a-a8b5-f42b0883bd24,0,Fluconazole 750mg,Fluconazole,Comprimé,750mg,,
+80a43c66-adc2-4e5a-b540-80a599c9eebe,0,Fludrocortisone 100µgs,Fludrocortisone,Comprimé,100µgs,,
+ddeafac5-c9dd-40fc-ab63-dcbea4885bf1,0,"Flumazénil 0,5mg/5ml",Flumazenil,Injection,"0,5mg/5ml",,
+8f1b8939-61c0-48b2-80da-b9a00c5d3d88,0,Fluoreceine sodique 10%,Fluorecein Sodium,Injection,10%,,
+96fec354-c939-4428-86a1-73737fdfc6c9,0,Fluorescéine 1%,Fluorescein,Collyre,1%,,
+d0d125dd-9181-47ec-96ad-871b7762897b,0,Fluorescéine 10%,Fluorescein,Injection,10%,,
+283630bf-4789-4345-9eee-b2b5e9d6fb52,0,"Fluorométholone 0,1%",Fluorometholone,Collyre,"0,10%",,
+9d7a114f-cf23-4352-97f5-1bf7941fc044,0,Fluoxétine 20mg /,Fluoxetine,Comprimé,20mg,,
+41d606ba-2b4b-4276-a789-c27e883ac0e7,0,Fluphénazine 25mg/ml 5ml,Fluphenazine,Injection,25mg/ml,,
+d3b0c30a-1416-4d1e-8fae-6825d050eb9c,0,Folinate de calcium 100mg/ml,Calcium Folinate,Injection,100mg/ml,,
+a0bc3fe0-f04f-46ca-abca-333403b9ff63,0,Formol 1g,Formalin,Comprimé,1g,,
+c57f2a44-3a60-41a9-9ead-6829595fd84a,0,Formol 40%,Formalin,Solution,40%,,
+161e913f-7526-4276-bfdf-7a8298af184b,0,Framycétine+dexaméthasone 1%,Framycetin,Goutte auriculaires,1%,,
+edeefc76-f410-4cad-a497-a21959621069,0,Furosémide 10mg,Furosemide,Comprimé,10mg,,
+061cae6b-fc45-4b31-ac30-1d00c17ca32f,0,Furosémide 10mg/ml,Furosemide,Ampoule,10mg/ml,,
+27f5aca4-32ab-4193-b33a-f6586e643f61,0,Furosémide 10mg/ml 2ml,Furosemide,Injection,10mg/ml,,
+d556687a-9e8b-4d0f-8625-57635ff08bb1,0,Furosémide 20mg/2ml,Furosemide,Ampoule,20mg/2ml,,
+32bf5c95-81fe-42fb-8020-6961908be3a3,0,Furosémide 20mg/5ml,Furosemide,Sirop,20mg/5ml,,
+eecd4761-3f87-4711-bf7d-0630fb38241b,0,Furosemide 40mg sous blister,Furosemide,Comprimé,40mg,,
+f7356169-c455-4c7b-a8ba-5be6dc1ca844,0,Gamma (difeuines) FL/250ml,Gamma Syrup (Difeuines),Sirop,FL/250ml,,
+ff343e59-1f62-4965-be6e-9c37151154d8,0,Gatifloxacine 400mg,Gatifloxacin,Comprimé,400mg,,
+bf26ca37-c684-40bf-a375-1f00decfe33b,0,Gélatine fluide modifiée 500ml,Modified Fluid Gelatin,Injection,500ml,,
+00de8bbd-d69e-474c-b331-e6de3360fc40,0,Gentamicine 10mg/ml,Gentamicin,Injection,10mg/ml,,
```

Comment:
@lauravignoli Not sure if this is a typo or not; should "Gentamicine" read  "Gentamycine"? please see line 383 to 385 as well. Note: this drug is also spelled Gentamicine in the excel sheet